aria-practices.html

```diff
@@ -771,39 +772,31 @@ <h3>Combo Box</h3>
       </p>
       <p>
         When constructing a widget that is both visually compact and enables users to choose one value from a set of discrete values, often either a <a href="#Listbox">listbox</a> or <a href="#menubutton">menu button</a> is simpler to implement and use.
-        One feature of combobox that distinguishes it from both listbox and menu button is that the value of the combobox is presented in an edit field.
+        One feature that combobox supports that distinguishes it from both listbox and menu button is that the value of the combobox can be presented in an editable field.
         Thus, the combobox gives users one function that both listbox and menu button lack, namely the ability to select some or all of the value for copying to the clipboard.
         One feature that distinguishes both combobox and menu button widgets from listbox widgets is their ability to provide an undo mechanism.
         In many implementations, users can navigate the set of allowed values in a combobox or menu and then decide to revert to the value the widget had before navigating by pressing <kbd>escape</kbd>.
         In contrast, navigating a listbox immediately changes its value, and escape does not provide an undo mechanism.
       </p>
-      <p class="note">
-        The options for a combobox to popup a grid, tree, or dialog were introduced in ARIA 1.1.
-        Changes made in the ARIA 1.1 specification also add support for a code pattern that enables assistive technologies to present the textbox and popup as separately perceivable elements.
-        both ARIA 1.0 and 1.1 patterns are described in the following sections.
-        While using the ARIA 1.1 pattern is recommended as soon as assistive technology support is sufficient,
-        there are no plans to deprecate the ARIA 1.0 pattern.
-      </p>
 
       <section class="notoc">
         <h4>Examples</h4>
```

Comment:
There is currently no example of a combobox that is not editable. We should add such an example.

Reply:
Agree. Please add thoughts to this issue: https://github.com/w3c/aria-practices/issues/1026